src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
-public class ColorSupplier {
+import java.util.Random;
+
+public abstract class ColorSupplier {
```

Comment:
Remove the `abstract` keyword from the class declaration, as there is no need for this class to be abstract. 🧐